pkg/services/sqlstore/migrations/panel_mig.go

```diff
@@ -0,0 +1,16 @@
+package migrations
+
+import (
+	. "github.com/grafana/grafana/pkg/services/sqlstore/migrator"
+)
+
+func addDashboardPanelMigrations(mg *Migrator) {
+	mg.AddMigration("Add column panel_titles in dashboard", NewAddColumnMigration(DashboardV2, &Column{
+		Name: "panel_titles", Type: DB_NVarchar, Length: 255, Nullable: true,
```

Comment:
given a dashboard can contain thousands of panels, this seems a bit low

Reply:
Indeed, the intention is to use `DB_LongText` ultimately (and adjust the `Length` value as well). However, setting `Type` to `DB_LongText` was making some CI tests fail for some reason. I decided to set it to `DB_NVarchar` temporarily until I could figure out what the issue is: [Change panel titles type to NVarchar temporarily](https://github.com/grafana/grafana/pull/74722/commits/06a36ca8c9704387d472a4e0a1e793b6088e9af8)

Reply:
Can you try to remove the length from the `DB_LongText` column? This adds the following in the add column statement:
https://github.com/grafana/grafana/blob/440f9a6ffbf8da30b069cee5c7e071b866c81f92/pkg/services/sqlstore/migrator/mysql_dialect.go#L76-L89
but the [LONGTEXT](https://dev.mysql.com/doc/refman/8.0/en/string-type-syntax.html) type does not expect length (like other types eg VARCHAR  do)

Reply:
however, we need also to take into account the [MySQL limits](https://dev.mysql.com/doc/refman/8.0/en/column-count-limit.html#row-size-limits) given that the specific table has already a TEXT column (`data`)

Reply:
That's what I did originally but the tests were failing then as well. Will try it again and look into the tests a bit.